grafana/image.bzl

```diff
@@ -52,8 +52,6 @@ def grafana_image(name, datasources, dashboards, plugins = [], env = {}, visibil
         # Dashboard files must be writable for entrypoint.sh.
         mode = "0o666",  # octal
         package_dir = "/var/lib/grafana/dashboards/",
-        # If strip prefix is not set, directory structure is flattened which we don't want.
```

Comment:
This may be a breaking change, maybe we could hide this behind a flag ? It may not change anything in this case though?